src/algebra/roption.lean

```diff
@@ -0,0 +1,93 @@
+/-
+Copyright (c) 2019 Joe Cool. All rights reserved.
```

Comment:
Copyright should be the first author.

Reply:
thank you for your information. I changed it